apps/docs/sidebars.js

```diff
@@ -2314,6 +2314,11 @@ collapsible: false,
           id: 'object-storage/how-to/delete',
           label: 'Delete object storage service',
         },
+        {
+          type: 'doc',
+          id: 'object-storage/how-to/curl-file',
+          label: 'How to Download File from a Private Object Storage Bucket with cURL',
```

Comment:
pardon za hnidopisstvi, ale smazala bych How to a nedavala tomu title case kdyz jinde neni (at je to aspon v ramci jedny page stejny) 
![image](https://github.com/user-attachments/assets/f9201350-429c-44ee-9656-7dbe71fde91a)